CMakeLists.txt

```diff
@@ -162,13 +162,15 @@ write_basic_package_version_file(
   VERSION ${MANIFOLD_VERSION}
   COMPATIBILITY SameMajorVersion
 )
-install(EXPORT manifoldTargets DESTINATION ${CMAKE_INSTALL_DATADIR}/manifold)
+install(EXPORT manifoldTargets
```

Comment:
What are the files being installed with this install call?

Reply:
The DESTINATION is ${CMAKE_INSTALL_LIBDIR}/cmake/manifold, so lib/cmake/manifold.  That seems to be where CMake is looking for these files,  at least on Linux:

https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure

I'm open to a better specification for where to place the files that will be better for cross-platform scenarios - the EXPORT feature is new to me, so I'm not properly conversant with best practices yet.

Reply:
Do you want to try to merge this PR, or just do this as part of your larger reorganization?

Reply:
I'm game either way - if you think there's a good chance of the larger reorg getting merged there's probably no particular need for this, but if that's problematic this would be helpful.

Reply:
Let's see how the other does, since this isn't passing the CI anyway.